AutomationTests/Malafi.Tests/Features/DocumentType.feature

```diff
@@ -0,0 +1,18 @@
+﻿Feature: DocumentType
+
+A short summary of the feature
+
+
+Background: Login to Internal Portal and click on Malafi Link
```

Comment:
Please change this line to be:
Common steps till reach to Document Types listing page